### DISCUSSION TITLE
Meta Conversions API

### DISCUSSION THREAD

**Discussion by FJPlusOneX** [saleor/saleor#12873] [General] [answered]
Hi there,

I am looking for a method to install Meta's Conversions API in Saleor. Here is a link to understand what the Meta Conversions API is: https://www.facebook.com/business/help/2041148702652965?id=818859032317965

This allows for the tracking of user events directly from the web platform (server based) as opposed to relying exclusively on browser events. Browser events are not as accurate as server based events, so the Conversions API is critical for recording accurate data. Examples of the types of events that are recorded are product views, add to carts, checkouts, and purchases - a long with all the item parameters.

Platforms such as Shopify, Magento, WooCommerce, and Wix all have integrations available for Meta Conversions API - however I can't seem to find any resources for implementation on Saleor. Is there a solution in Saleor for the installation of the Meta Conversions API?

Any knowledge on this would be much appreciated.

With thanks,
F

**Comment by timuric** [comment, marked as answer]
@FJPlusOneX the difference between Shopify and woocommerce is that Saleor is designed to be haedless, it is completely decoupled from front-end and it is up to you to build your theme/storefront the way you want.

Frameworks such as next.js and remix provide convenient way to enable server side tracking.
